### PR TITLE
[SETUP] French translation of missing elements

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- After selecting "French" during setup, 2 pages are still en english : version status and Licence T&C

install2
install

## Proposed changes

- Translation in French of the above mentionned 2 pages.

VirtualBox_ReactOS_30_08_2020_20_08_15
VirtualBox_ReactOS_30_08_2020_16_39_28

### CODE REVIEW

**Comment by KyleKatarn31** [comment]
Cc @reactos/lang-french @sdever

**Comment by KyleKatarn31** [comment]
Ready for review.

File: base/setup/usetup/lang/fr-FR.h [hunk 1]
```diff
@@ -192,63 +192,63 @@ static MUI_ENTRY frFRIntroPageEntries[] =
...
-        "evaluation and testing purposes and not as your daily-usage OS.",
+        "\202valuation et test, pas pour un usage quotidien.",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         6,
         15,
-        "Backup your data or test on a secondary computer if you attempt",
+        "Sauvez vos donn\202es ou testez sur un ordinateur secondaire",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:223]:
```suggestion
        "Sauvegardez vos donn\202es ou testez sur un ordinateur secondaire",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 2]
```diff
@@ -279,77 +279,77 @@ static MUI_ENTRY frFRLicensePageEntries[] =
     {
         8,
         8,
-        "The ReactOS System is licensed under the terms of the",
+        "La licence du syst\212me ReactOS System est sous les termes de",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:282]:
```suggestion
        "La licence du syst\212me ReactOS est sous les termes de",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 3]
```diff
@@ -279,77 +279,77 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
-        "This software comes with NO WARRANTY or restrictions on usage",
+        "Ce logiciel vient SANS GARANTIE ou restriction d'usage",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         16,
-        "save applicable local and international law. The licensing of",
+        "sous lois lois locales ou intenationales applicables.",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:331]:
```suggestion
        "sous lois locales ou intenationales applicables.",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 4]
```diff
@@ -370,21 +370,21 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
-        "There is NO warranty; not even for MERCHANTABILITY or",
+        "Il n'y a AUCUNE garantie; pas même sur la QUALITE ou ",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         26,
-        "FITNESS FOR A PARTICULAR PURPOSE",
+        "l'ADEQUATION POUR UN USAGE",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:387]:
```suggestion
        "l'ADEQUATION POUR UN USAGE EN PARTICULIER",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 5]
```diff
@@ -370,21 +370,21 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
-        "This is free software; see the source for copying conditions.",
+        "Ce logiciel est gratuit. Voir les sources pour les conditions de copie.",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         25,
-        "There is NO warranty; not even for MERCHANTABILITY or",
+        "Il n'y a AUCUNE garantie; pas même sur la QUALITE ou ",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:380]:
```suggestion
        "Il n'y a AUCUNE garantie; pas même sur la QUALITE ou",
```

**Comment by KyleKatarn31** [comment]
All comments from @HBelusca taken into account (thanks !) and screenshots updated accordingly.

**Comment by gonzoMD** [comment]
I'd suggest that you add some line breaks to avoid that the lines reach till the right end of the screen.

**Comment by KyleKatarn31** [comment]
> I'd suggest that you add some line breaks to avoid that the lines reach till the right edge of the screen.

Done !

**Comment by KyleKatarn31** [comment]
@HBelusca @gonzoMD @sdever : ready to go ?

**Comment by KyleKatarn31** [comment]
Would it be possible to merge now that no more comments are open ?

File: base/setup/usetup/lang/fr-FR.h [hunk 6]
```diff
@@ -192,63 +192,63 @@ static MUI_ENTRY frFRIntroPageEntries[] =
...
-        "ReactOS is in Alpha stage, meaning it is not feature-complete",
+        "ReactOS est en version Alpha, ce qui signifie qu'il n'est pas complet",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         6,
         12,
-        "and is under heavy development. It is recommended to use it only for",
+        "et est en cours de d\202veloppement. A n'utiliser que pour",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:209]:
```suggestion
        "et est en cours de d\202veloppement. Il est recommand\202 de ne l'utiliser",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 7]
```diff
@@ -192,63 +192,63 @@ static MUI_ENTRY frFRIntroPageEntries[] =
...
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         6,
         13,
-        "evaluation and testing purposes and not as your daily-usage OS.",
+        "\202valuation et test, pas pour un usage quotidien.",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:216]:
```suggestion
        "que pour \202valuation et test, pas pour un usage quotidien.",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 8]
```diff
@@ -279,112 +279,126 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
+        "conditions de copie.",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
-        26,
-        "FITNESS FOR A PARTICULAR PURPOSE",
+        27,
+        "Il n'y a AUCUNE garantie; pas m\210me sur la QUALITE ou",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:394]:
```suggestion
        "Il n'y a AUCUNE garantie; pas m\210me sur la QUALIT\220 ou",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 9]
```diff
@@ -279,112 +279,126 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        8,
+        28,
+        "l'ADEQUATION POUR UN USAGE EN PARTICULIER",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:401]:
```suggestion
        "l'AD\220QUATION POUR UN USAGE EN PARTICULIER",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 10]
```diff
@@ -279,112 +279,126 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
-        "This software comes with NO WARRANTY or restrictions on usage",
+        "Ce logiciel vient SANS GARANTIE ou restriction d'usage",
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         16,
-        "save applicable local and international law. The licensing of",
+        "sous lois locales ou intenationales applicables.",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:331]:
```suggestion
        "sous les lois locales ou internationales applicables.",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 11]
```diff
@@ -279,112 +279,126 @@ static MUI_ENTRY frFRLicensePageEntries[] =
     {
         8,
         8,
-        "The ReactOS System is licensed under the terms of the",
+        "La licence du syst\212me ReactOS est sous les termes de",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:282]:
```suggestion
        "Le syst\212me ReactOS est distribu\202 sous les termes de",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 12]
```diff
@@ -279,112 +279,126 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         9,
-        "GNU GPL with parts containing code from other compatible",
+        "GNU GPL avec des parties contenant du code sous",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:289]:
```suggestion
        "la licence GNU GPL avec des parties contenant du code sous",
```

File: base/setup/usetup/lang/fr-FR.h [hunk 13]
```diff
@@ -279,112 +279,126 @@ static MUI_ENTRY frFRLicensePageEntries[] =
...
         TEXT_STYLE_NORMAL,
         TEXT_ID_STATIC
     },
     {
         8,
         10,
-        "licenses such as the X11 or BSD and GNU LGPL licenses.",
+        "autres licenses compatibles telles que X11, BSD et GNU LGPL.",
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h:296]:
```suggestion
        "d'autres licences compatibles telles que X11, BSD et GNU LGPL.",
```

**Review by HBelusca — COMMENTED** [review]
Small adjustments in the french translation. Cc @KyleKatarn31

**Comment by KyleKatarn31** [comment]
Thank you @HBelusca. All comments taken into account. Ready for merge.

**Comment by KyleKatarn31** [comment]
Thanks @HBelusca

**Pull request merged**